app/views/users/show.html.erb

```diff
@@ -145,14 +145,16 @@
       </div>
     </div>
     <%= render "articles/user_metadata", context: "profile" %>
-    <div class="profile-dropdown">
-      <button id="user-profile-dropdown">
-        <%= image_tag("three-dots.svg", class: "dropdown-icon", alt: "Toggle dropdown menu") %>
-      </button>
-      <div id="user-profile-dropdownmenu" class="profile-dropdownmenu">
-        <a href="/report-abuse?url=https://dev.to/<%= @user.username %>">Report Abuse</a>
+    <% unless @user == current_user %>
```

Comment:
Perhaps we could include a TODO comment to move this conditional down to only the report link if more features are added to this dropdown? I'm not sure about your preference when it comes to TODO comments, but I tend to like them.

Reply:
Good idea. I'll add it